crates/db_schema/src/source/post.rs

```diff
@@ -58,6 +58,7 @@ pub struct Post {
   pub url_content_type: Option<String>,
   /// An optional alt_text, usable for image posts.
   pub alt_text: Option<String>,
+  pub scheduled_time: Option<DateTime<Utc>>,
```

Comment:
Pry add comments here, that `None` means published immediately. 

Also `publish_time` or `scheduled_publish_time` might be a better name for this column.

Reply:
Added comment and renamed column.